deploy-cockroachdb-on-microsoft-azure.md

```diff
@@ -103,35 +103,36 @@ Microsoft Azure offers fully-managed load balancing to distribute traffic betwee
 
 Locally, you'll need to [create the following certificates and keys](create-security-certificates.html):
 
-- A certificate authority (CA) key pair (`ca.cert` and `ca.key`)
+- A certificate authority (CA) key pair (`ca.crt` and `ca.key`)
```

Comment:
Based on the rest of your updates, we should use the `.cert` filename convention here.